platforms/qrb2210-rb1/partitions.conf

```diff
@@ -21,17 +21,22 @@
 
 # This is physical partition 0
 --partition --name=xbl_a --size=3584KB --type-guid=DEA0BA2C-CBDD-4805-B4F9-F428251C3E98 --filename=xbl.elf
---partition --name=xbl_b --size=3584KB --type-guid=77036CD4-03D5-42BB-8ED1-37E5A88BAA34 --filename=xbl.elf
+--partition --name=xbl_b --size=3584KB --type-guid=7A3DF1A3-A31A-454D-BD78-DF259ED486BE --filename=xbl.elf
```

Comment:
Please describe, how / why did you change GUIDs for _b partitions. I think even in the UEFI usecase (qcs6490 / qcs9100) all _b partitions have the same value.

Reply:
=> commit message

Reply:
Done